### PR TITLE
Support supplying image url with path component

### PR DESCRIPTION
**Pull request by sbly**
Necessary for [this ticket](https://themuse.atlassian.net/browse/SVC-209)

### CODE REVIEW

**Comment by ian-axelrod** [comment]
Do we want to get tests working for this?

**Comment by sbly** [comment]
@ian-axelrod I'm working on getting tests set up locally (I get to use Vagrant again!). I don't think it's worthwhile to get a CI setup going

**Comment by sbly** [comment]
Alright tests pass locally. Only a single test fails, which is due to a timeout error, and it also fails on master so it's just flaky.

**Comment by ian-axelrod** [comment]
Ok, as long as tests pass in `pilbox-docker` pointing to the new repo, that's fine.

File: pilbox/app.py [hunk 1]
```diff
@@ -138,7 +138,7 @@ def __init__(self, **kwargs):
         tornado.web.Application.__init__(self, self.get_handlers(), **settings)
 
     def get_handlers(self):
-        return [(r"/", ImageHandler)]
+        return [(r"/(.*)", ImageHandler)]
```
Comment (ian-axelrod) [inline pilbox/app.py:141]: Is this the only functional change? If so, can we not bother making other changes to the fork? I hate it when forks diverge substantially from the original version, as that makes merging upstream more tedious.

**Review by ian-axelrod — CHANGES_REQUESTED** [review]

File: pilbox/app.py (hunk 1 above)
Comment (sbly) [reply pilbox/app.py:141]: I don't believe this is going to get merged upstream. Also, I'm not sure exactly what you're asking here?

Comment (ian-axelrod) [reply pilbox/app.py:141]: The other changes you made just appear to be refactors. This particular change is the one that satisfies the ticket requirements, correct?

**Comment by ysimonson** [comment]
This is a pretty big divergence from pilbox upstream, which means we'll merge the latest updates from upstream even more rarely. Before we go down this route, let's discuss tomorrow and see what other options there might be?

File: pilbox/app.py (hunk 1 above)
Comment (sbly) [reply pilbox/app.py:141]: This is the only change I made. I didn't do any refactoring

**Comment by sbly** [comment]
@ysimonson OK, happy to discuss

File: pilbox/app.py (hunk 1 above)
Comment (ian-axelrod) [reply pilbox/app.py:141]: ?? But I see 'Two files modified' and more than one commit.

Comment (sbly) [reply pilbox/app.py:141]: I had to make these changes to get the code to work. I couldn't just change this one line. Happy to discuss on slack if you want me to go over the changes

**Comment by sbly** [comment]
@ysimonson It looks like this is the solution that we're going with. Good to merge?

File: pilbox/app.py (hunk 1 above)
Comment (sbly) [reply pilbox/app.py:141]: Is this good?

**Review by ysimonson — APPROVED** [review]

**Review by ian-axelrod — APPROVED** [review]

**Pull request merged**